frontend/src/scenes/feature-flags/FeatureFlagReleaseConditions.tsx

```diff
@@ -454,7 +455,12 @@ export function FeatureFlagReleaseConditions({
                                     <h3 className="l3">Release conditions</h3>
                                     <div className="text-muted mb-4">
                                         Specify the {aggregationTargetName} to which you want to release this flag. Note
-                                        that condition sets are rolled out independently of each other.
+                                        that condition sets are rolled out independently of each other; and cohort-based
+                                        targeting{' '}
+                                        <Link to="https://posthog.com/docs/data/cohorts#can-you-use-a-dynamic-behavioral-cohort-as-a-feature-flag-target">
+                                            does not support dynamic behavioral cohorts
+                                        </Link>
+                                        , so those cohorts will not appear as filter options.
```

Comment:
TODO make this text less verbose probably – it's only two sentences but one is a bit of a run-on.